CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ A more detailed list of changes is available in the corresponding milestones for
   - **license_path** condition: do not cause an ERROR on families lacking a license file. (issue #3201)
 
 ### New Checks
+  - **[com.google.fonts/check/cjk_sparse_glyphs]:** Warn users if there are less than 40 CJK glyphs in a font.
```

Comment:
Please always end the `CHANGELOG.md` entries with an issue number within parentheses (or, in this case, the PR number). So here it would be:

- **[com.google.fonts/check/cjk_sparse_glyphs]:** Warn users if there are less than 40 CJK glyphs in a font. (PR #3214)

Reply:
the reasoning behind this practice is to make it easy for someone interested to find any further info that may have been provided in discussions when the change was implemented.

Reply:
Thanks. Will implement this going forward.